src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/DummyNode.cs

```diff
@@ -24,12 +25,26 @@ public void CustomizeView(DummyNode model, Dynamo.Controls.NodeView nodeView)
                 Width = 66.0,
                 Height = 66.0,
                 Stretch = Stretch.UniformToFill,
+                HorizontalAlignment = HorizontalAlignment.Center,
+                VerticalAlignment = VerticalAlignment.Center,
                 Source = new BitmapImage(new Uri(src, UriKind.Relative))
             };
             RenderOptions.SetBitmapScalingMode(dummyNodeImage, BitmapScalingMode.HighQuality);
 
             nodeView.inputGrid.Children.Add(dummyNodeImage);
             model.Warning(model.GetDescription(), true);
+
+            UIElement child = nodeView.grid.FindName("zoomGlyphsGrid") as UIElement;
+            nodeView.grid.Children.Remove(child);
+
+            child = nodeView.grid.FindName("nodeColorOverlayZoomIn") as UIElement;
+            nodeView.grid.Children.Remove(child);
+
+            child = nodeView.grid.FindName("nodeBorder") as UIElement;
+            nodeView.grid.Children.Remove(child);
+
+            child = nodeView.grid.FindName("GlyphStackPanel") as UIElement;
+            nodeView.grid.Children.Remove(child);
```

Comment:
I am debating with myself if this is the best way to do this but I think this is OK as long as this is the only node where this kind of exception needs to be added.

Reply:
> I am debating with myself if this is the best way to do this but I think this is OK as long as this is the only node where this kind of exception needs to be added.

I wasn't quite sure if that was the way to go, when I found out how the paperclip icon was implemented I thought it was a bit hacky. 

Initially, I went a different route through the HomeWorkspaceViewModel checking if the node was of type DummyNode to then suppress the glyphs. However, the glyph container visibility already works quite heavily, and I didn't want to overcomplicate it. Furthermore, I was making a lot more changes to the code compared to removing the glyph container only in the DummyNode.